src/app/[locale]/category/[category]/page.tsx

```diff
@@ -1,7 +1,11 @@
-const category = () => {
+import { fetchRecipes } from "@/app/actions/actions";
+import Recipes from "@/components/Recipe/Recipe";
+
+const category = async () => {
```

Comment:
`src/app/[locale]/categories/[category]` maybe? if on url **page pl/categories** will be more categories to choose